assets/js/base/components/payment-methods/saved-payment-method-options.js

```diff
@@ -145,31 +145,23 @@ const SavedPaymentMethodOptions = ( { onChange } ) => {
 	}, [
 		customerPaymentMethods,
 		updateToken,
-		selectedToken,
+		activeSavedToken,
+		activePaymentMethod,
 		setActivePaymentMethod,
 		setPaymentStatus,
 		standardMethods,
 	] );
 
-	// In the editor, show `Use a new payment method` option as selected.
-	const selectedOption = isEditor ? '0' : selectedToken + '';
-	const newPaymentMethodOption = {
-		value: '0',
-		label: __( 'Use a new payment method', 'woo-gutenberg-product-blocks' ),
-		name: `wc-saved-payment-method-token-new`,
-	};
 	return currentOptions.current.length > 0 ? (
 		<RadioControl
 			id={ 'wc-payment-method-saved-tokens' }
-			selected={ selectedOption }
-			onChange={ updateToken }
-			options={ [ ...currentOptions.current, newPaymentMethodOption ] }
+			selected={ activeSavedToken }
+			onChange={ ( ...args ) => {
+				updateToken( ...args );
+			} }
```

Comment:
Why is this wrapped in another function instead of just `onChange={ updateToken }`?

Reply:
Good suggestion. @haszari do you remember if there was a particular reason why this was done this way?

Reply:
No idea, I'm always a fan of less functions, less indirection so this is good advice from @nerrad  IMO 😸. Maybe there used to be something else happening inside the inline func (which has since been removed).